### ISSUE TITLE
Make neural network training

### ISSUE THREAD

**Issue by Gabrielle-Ribeiro** [deeplearningunb/stonks#30]
**Describe the solution you'd like**

Apply callback functions and train the neural network.

**Additional context**

Read about callback functions in [Keras documentation](https://keras.io/callbacks/).

**Checklist**
- [x] The issue has a meaningful name.
- [x] The issue has a meaningful description.
- [ ] The issue has screenshots when needed.
- [x] The issue has labels.
